### PR TITLE
Added CCS init to BAE LP

### PR DESCRIPTION
**Pull request by beknazaresenbek**

### CODE REVIEW

File: charts/business-api-ecosystem/templates/biz-ecosystem-logic-proxy/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```
Comment (dwendland) [inline charts/business-api-ecosystem/templates/biz-ecosystem-logic-proxy/configmap.yaml:1]: That CM should be created optional

File: charts/business-api-ecosystem/values.yaml [hunk 2]
```diff
@@ -683,6 +683,21 @@ bizEcosystemLogicProxy:
   # -- Base app path of logic proxy (for versions < 8.1.0: /business-ecosystem-logic-proxy, for versions >= 8.1.0: /opt/business-ecosystem-logic-proxy)
   basePath: /business-ecosystem-logic-proxy
 
+  ccs:
+    # -- Endpoint of the CCS, e.g. http://credentials-config-service:8080
+    endpoint: ""
```
Comment (dwendland) [inline charts/business-api-ecosystem/values.yaml:688]: Maybe this should be commented out to make sure that conditions are not fulfilled by default, like here: https://github.com/i4Trust/helm-charts/blob/main/charts/pdc-portal/values.yaml#L147

Comment (beknazaresenbek) [reply charts/business-api-ecosystem/values.yaml:688]: Empty string has the same effect, and there won't be warnings about non-existing property

File: charts/business-api-ecosystem/templates/biz-ecosystem-logic-proxy/configmap.yaml (hunk 1 above)
Comment (beknazaresenbek) [reply charts/business-api-ecosystem/templates/biz-ecosystem-logic-proxy/configmap.yaml:2]: Ok, I've updated it

File: charts/business-api-ecosystem/values.yaml (hunk 2 above)
Comment (dwendland) [reply charts/business-api-ecosystem/values.yaml:688]: ok

**Review by dwendland — APPROVED** [review]

**Pull request merged**